<ui version="4.0">
  <class>ChatDialog</class>
  <widget class="QDialog" name="ChatDialog">
    <property name="geometry">
      <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Sync Demo</string>
    </property>
    <!-- start of root layout -->
    <layout class="QVBoxLayout">
      <property name="spacing">
        <number>6</number>
      </property>
      <!-- start of head bar item -->
      <item >
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <layout class = "QVBoxLayout" >
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel" name="infoLabel">
                <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="prefixLabel">
                <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QPushButton" name="setButton">
            <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
              <string>Settings</string> 
            </property>
            <property name="minimumWidth">
              <number>150</number>
            </property>
            <property name="maximumWidth">
              <number>150</number>
            </property>
            </widget>
          </item>
        </layout>
      </item>
      <!-- end of head bar item -->
      <!-- start of display area -->
      <item>
        <layout class = "QHBoxLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class = "QListView" name="listView">
              <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QVBoxLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QGraphicsView" name="treeViewer">
                <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                </property>
                </widget>
              </item>
              <item>
                <widget class="QTextEdit" name="textEdit">
                  <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                  </property>
                  <property name="minimumWidth">
                    <number>600</number>
                  </property>
                  <property name="readOnly">
                    <bool>true</bool>
                  </property>
                </widget>
              </item>
              <!-- start of input line widget -->
              <item>
                <layout class="QHBoxLayout">
                  <property name="margin">
                    <number>0</number>
                  </property>
                  <property name="spacing">
                    <number>6</number>
                  </property>
                  <item>
                    <widget class="QLabel" name="label">
                      <property name="text">
                        <string>Message:</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="lineEdit" />
                  </item>
                </layout>
              </item>
              <!-- end of input line widget -->
            </layout>
          </item>
        </layout>
      </item>
      <!-- end of display area -->
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
	
